.github/workflows/deploy_to_environment.yml

```diff
@@ -71,26 +52,34 @@ jobs:
         if: contains(inputs.env, 'impl') || contains(inputs.env, 'prod')
         run: |
           ./scripts/rds-snapshot-app-db
-      - name: Run migrations
+      - name: Run database migrations
         env:
-          AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}
           ECR_REGISTRY: ${{ secrets.ECR_REGISTRY }}
           ECR_REPOSITORY: easi-db-migrate
-          LAMBDA_VERSION: ${{ inputs.lambda_version }}
-          ENV: ${{ inputs.env }}
+          NEW_IMAGE_TAG: ${{ env.GIT_HASH }}
+          TASK_FAMILY: "db-migrate-${{ inputs.env }}"
```

Comment:
![image](https://github.com/CMSgov/easi-app/assets/15203744/b3e85609-9d49-4ed7-80f7-5da90888bea2)

@azeez-salu I think this needs to be

```suggestion
          TASK_FAMILY: "easi-app-db-migrate-${{ inputs.env }}"
```